arrow-libs/core/arrow-meta/src/main/java/arrow/common/utils/AbstractProcessor.kt

```diff
@@ -92,13 +91,7 @@ abstract class AbstractProcessor : KotlinAbstractProcessor(), ProcessorUtils, Ko
     else -> knownError("Unsupported @documented $kind")
   }
 
-  private fun processDocs(roundEnv: RoundEnvironment): Unit =
```

Comment:
is this on purpose? If so, wouldn't be possible to delete the entire class instead?

Reply:
This is on purpose. The `Docs` feature is no longer supported, but the `AbstractProcessor` is still in use until `kapt` for Optics is replaced by Arrow Meta.